### PR TITLE
More names

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- More Names
- Adjustments of name inheritance (wip)
-

Review please

### CODE REVIEW

File: src/main/resources/data/minecolonies/citizennames/dutch.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "Van de Groenendike", "De Ha", "De Hainnuires", "De Ham", "De Harlebecca", "Heaumes", "Hegde", "Henueres", "De Herines", "De Heripont", "VanHertsberghe", "Hoeft", "Hoevet", "De Hopscota", "De Hostbors", "De Hoves", "Hovet", "Hubrecht", "Hubrechte", "Hubrechtes", "LeHungre", "Die Jonghe", "Joris", "Kerstiaen", "Lam", "Lammain", "De Lampernessa", "De Lampernesse",
+    "Langore", "De Larbecha", "De Laterbuse", "De LeHeie", "De Lembecha", "De Lens", "De Longa", "Roade", "Longerue", "Loots", "FiuLoten", "FiuLotin", "FiliusLotinide", "Maisnil", "De leMalte", "De Marcha", "De leMarcha", "Margaretenzoene", "De Marvis", "FiliusMeidini", "De Merlemont", "De Morlenwes", "De Nemore", "Van de Niewerport", "De NivellaOchons", "Oems", "De Ostiskirke",
+    "Paskin", "DictusParde", "RoprePauwelszoene", "DictusPerde", "Topre", "Piel", "Pilsart", "DictusPriem", "DictaPriems", "DictusPriems", "ProvensinQuade", "Uvima", "LeRaed", "De Ravia", "FiliusRavoini"
```
Comment (uecasm) [inline src/main/resources/data/minecolonies/citizennames/dutch.json]: Is that intended to have several PascalCase names like "DictusParde" and not "Dictus Parde" or "Dictus-Parde"?

File: src/main/resources/data/minecolonies/citizennames/english.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	"Uganus", "Abraham", "Adam", "Alueredus", "Dilectus", "Dunewaldus","Ernulfus", "Gerardus", "Gundulfus", "Herewaldus", "Herveus", "Hillarius", "Hubertus", "Jacobus", "Jocelinus", "Manaser", "Morganus", "Nigellus", "Rodbertus", "Ruelen", "Sampson"
+
+  ],
+  "female_firstname": [
+    "Alice", "Matilda", "Agnes","Emma", "Gunilda", "Gunilde", "Gunnilda", "Margery", "Wimar", "Wimarc", "Wymarc", "Wymarcha", "SaraSarah", "Amice", "Amicia", "Avice", "Avicia", "Beatrice", "Christiana", "Christina", "Cristiana", "Cristina", "De nise", "Joan", "CeciliaCecily", "Gillian", "IsabelIsabella", "Larica", "Clarice", "Claricia","Emelota","Eva", "Helen", "Mabel", "Margaret",
+	"SibylSybil", "Alina", "Constance", "Custance","Ede", "Lota","Edith", "Julian", "Juliana", "Katherine", "Maisenta", "Scolastica", "Seilda", "Agatha", "Albreda", "Ascelina", "Ascelota", "Auwenilda", "Cenota", "Clementia","Eda","Ede", "Lina","Elena","Eustachia", "Gillota", "Goda", "Gundreda", "Heloise", "Hodierna", "Idonea", "Imeina", "Isolda", "Mariota", "Olive", "Oresia", "Osanna", "Pelagia",
```
Comment (uecasm) [inline src/main/resources/data/minecolonies/citizennames/english.json]: "SibylSybil"?

File: src/main/resources/data/minecolonies/citizennames/english.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	"Petronilla", "Philippa", "Roesia", "Rosamund", "Rose", "Sabina", "Susan", "Swanekil", "Swanekilda", "Theophania"
+  ],
+  "surnames": [
+   "Bangornensis", "Bathoniensis", "Bigod", "Biseth", "Capellanus", "Cycestrensis", "De Bende", "Nges", "De Bende", "Ngis", "De Bercherola", "De Bohun", "De Bude", "Forde", "De Evereus", "De Gloucestria", "De Haya", "De Hesdine", "De Jureio", "De Laceio", "De Landr'", "De Langforde", "De Lettrintone", "De Londonia", "De Mech'", "De Peke", "De Pennune", "De Saloppesburia", "De Sancto", "Hyllario", "De Canus",
```
Comment (uecasm) [inline src/main/resources/data/minecolonies/citizennames/english.json]: "De Landr'" (stray apostrophe)?

File: src/main/resources/data/minecolonies/citizennames/norse.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "Skáld", "Austmaðr", "Inn halti", "Inn hávi", "Hólmgongu", "Inn mikli inn mjóvi", "Inn ríki", "Inn skjálgi", "Vikingr in auða", "Dala", "Inn fagri", "Hálmi", "Hjalti", "Kappi", "Kráka lambil", "Mjąksiglandi", "Tungu", "Þurs", "Inn austrœni", "Beigaldi bíldr", "Blákinn", "Blundr", "Bloōruskalli", "Brún", "Brækir", "Inn danski", "Dúfunef gjallandi", "Inn grái", "Gyðja Hafr", "Inn hamrammi", "Háleyski",
+    "heljarskinn", "Hestr holbarki", "Holmðr", "Hrogn", "Inn gamli", "Goði", "Inn hvíti", "Inn sterki", "Inn auðgi", "Inn rauði", "Inn spaki inn svarti", "Inn digri", "Inn rammi", "Viga", "Inn fróði", "Bersir", "Smiðr", "Inn goði", "Inn helgi", "Skáld", "Austmaðr", "Inn halti", "Inn hávi", "Hólmgongu", "Inn mikli inn mjóvi", "Inn ríki", "Inn skjálgi", "Vikingr in auða", "Dala", "Inn fagri", "Hálmi", "Hjalti",
+    "kappi", "Kráka lambil", "Mjąksiglandi", "Tungu", "Þurs", "Inn austrœni", "Beigaldi bíldr", "Blákinn", "Blundr", "Bloōruskalli", "Brún", "Brækir", "Inn danski", "Dúfunef gjallandi", "Inn grái", "Gyðja Hafr", "Inn hamrammi", "Háleyski", "Heljarskinn", "Hestr holbarki", "Holmðr", "Hrogn"
```
Comment (uecasm) [inline src/main/resources/data/minecolonies/citizennames/norse.json]: Intended some of these are "Inn" and some "inn"?  And some "Inn spaki inn <something else>"?

**Review by uecasm — COMMENTED** [review]
(Despite appearances, I still only skimmed the names; not a full proofread.)

File: src/main/resources/data/minecolonies/citizennames/dutch.json (hunk 1 above)
Comment (MotionlessTrain) [reply src/main/resources/data/minecolonies/citizennames/dutch.json]: I wasn't sure about that name at all, so I left it when I sent a fixed version to Ray. I guess Dictus-Parde would be the best alternative. I can't remember that anyone had a PascalCase name in Dutch history

File: src/main/resources/data/minecolonies/citizennames/norse.json (hunk 4 above)
Comment (MotionlessTrain) [reply src/main/resources/data/minecolonies/citizennames/norse.json]: I noticed in the Dutch ones that some names got split across multiple entries for some reason (which I already fixed). I'm not sure if the same could be happening to Norse as well.

**Comment by oddrobbia** [comment]
well i just copyed into a file and i tryed to organize the names it seems i not noticed this few errors that the others did ah i checked multiple times but i not noticed its good that others did

**Review by marchermans — APPROVED** [review]

**Pull request merged**